python_bindings/run_tutorial.sh

```diff
@@ -1,14 +1,10 @@
 #!/bin/bash
```

Comment:
Ditto comment on command line usage.

Reply:
Will do. (Both of these scripts really need to die and be replaced with something more generic. Subsequent PRs.)